docs/Makefile

```diff
@@ -0,0 +1,29 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
```

Comment:
`PYWREN_DOCS` should be defined by the user as an environment variable. If it isn't set, `make update` would error and explain. 

![screen shot 2017-08-22 at 1 46 13 pm](https://user-images.githubusercontent.com/7637700/29586654-5532e6f4-8740-11e7-8aac-a0a8a1efe4d7.png)